modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -1058,7 +1143,7 @@ events
 |Name |Value Type |Optional |Description
 |`adapt`
 |`CodeRef<(element: GraphElement) => AdapterDataType<PodsAdapterDataType> | undefined>`
-|no |
+|no | adapter to adapt element to data that can be used by Pod component. |
```

Comment:
also formatting issue in table
![Screenshot 2023-03-30 at 2 14 54 PM](https://user-images.githubusercontent.com/12692381/228746062-1d12ac84-78c8-465b-a504-1c0776692aeb.png)